user-guide/workspace-basics/index.njk

```diff
@@ -0,0 +1,36 @@
+---
+title: 3· Workspace basics
+---
+
+<h1 id="workspace-basics">Workspace basics</h1>
+<p class="main-paragraph">The Workspace is where you actually create your designs. You have an infinite canvas where you can work directly but you also have the ability to create and work inside artboards that will help you to create pages and exportation units.</p>
+
+<h3 id="viewport">The viewport</h3>
+<p>Surrounded by panels, header and toolbars, in the middle of the workspace, you can find the viewport. The viewport is the design area of a file page. It is practically infinite. If what you need is a frame with specific, limited dimensions, you can create an artboard.</p>
+<p><img src="/img/viewport.png" alt="the viewport" /></p>
+<h4>Navigate the viewport</h4>
+<p>Press Tab (your cursor) while moving your mouse to navigate the viewport. If you are using a trackpad you can do two finger scrolling.</p>
```

Comment:
This is by pressing space, not Tab, isn't it? And what does "your cursor" mean?

Reply:
dunnow who wrote that :P